Assets/_BForBoss/_Entities/Scripts/ScriptableObjects/HealthScriptableObject.cs

```diff
@@ -1,20 +1,12 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
 namespace Perigon.Entities
 {
-
-    public interface IHealth
-    {
-        public float Health { get; }
-    }
-    
     [CreateAssetMenu(fileName = "HealthProperties", menuName = "PerigonGames/Health", order = 1)]
-    public class HealthScriptableObject : ScriptableObject, IHealth
+    public class HealthScriptableObject : ScriptableObject
     {
-        [SerializeField] private float _health = 100f;
+        [SerializeField] private float _amount = 100f;
```

Comment:
I think it can still be called `health` no?

Reply:
Because the class is called healthscriptable, I found it weird if we named it health.... Since itll look like health.health;.

So health.amount makes more sense